### PR TITLE
gr: add

### PR DESCRIPTION
**Pull request by kou**

### CODE REVIEW

**Comment by elieux** [comment]
I'd rather download the tarball and extract it manually (see `noextract` and `prepare`) than to bring in the Git artillery.

**Comment by kou** [comment]
OK.
I've changed to use `tar.gz` instead of Git repository.

File: mingw-w64-gr/PKGBUILD [hunk 1]
```diff
@@ -0,0 +1,60 @@
...
+         "${MINGW_PACKAGE_PREFIX}-libpng"
+         "${MINGW_PACKAGE_PREFIX}-libtiff"
+         "${MINGW_PACKAGE_PREFIX}-qhull-git"
+         "${MINGW_PACKAGE_PREFIX}-qt5"
+         "${MINGW_PACKAGE_PREFIX}-zlib")
+makedepends=("git"
+             "${MINGW_PACKAGE_PREFIX}-cmake"
+             "${MINGW_PACKAGE_PREFIX}-gcc")
+options=("staticlibs" "strip" "!buildflags")
+source=("https://github.com/sciapp/gr/archive/v${pkgver}.tar.gz"
```
Comment (Alexpux) [inline mingw-w64-gr/PKGBUILD:27]: Download in named archive

File: mingw-w64-gr/PKGBUILD [hunk 2]
```diff
@@ -0,0 +1,60 @@
...
+      ../${_realname}-${pkgver} \
+      -DCMAKE_BUILD_TYPE=Release \
+      -DCMAKE_INSTALL_PREFIX=${MINGW_PREFIX} \
+      -G "MSYS Makefiles"
+  make
+}
+
+package() {
+  cd ${srcdir}/build-${CARCH}
+  make DESTDIR="${pkgdir}" install
```
Comment (Alexpux) [inline mingw-w64-gr/PKGBUILD:59]: If sources have license file then install it. See other our packages

File: mingw-w64-gr/PKGBUILD (hunk 1 above)
Comment (kou) [reply mingw-w64-gr/PKGBUILD]: I've changed to use `"${_realname}-${pkgver}.tar.gz::https://...`.

File: mingw-w64-gr/PKGBUILD (hunk 2 above)
Comment (kou) [reply mingw-w64-gr/PKGBUILD:59]: Done.

I'll send a pull request to install license file by `make install` to GR later.

**Pull request merged**